tools/opt/opt.cpp

```diff
@@ -623,16 +718,38 @@ OptStatus ParseFlags(int argc, const char** argv,
           return {OPT_STOP, 1};
         }
         optimizer->SetTargetEnv(target_env);
-      } else if (0 == strcmp(cur_arg, "--webgpu-mode")) {
```

Comment:
We generally don't like removing an option.  Are we sure it is not being used yet?

Reply:
I am not aware of anything that depends on this flag at the moment, other then the tests in the repo it isn't integrated with anything to my knowledge. The current implementation is incomplete at the moment and the spec isn't finalized.